task/rhacs-deployment-check/3.71/samples/pipelinerun.yaml

```diff
@@ -28,3 +29,6 @@ spec:
         resources:
           requests:
             storage: 500Mi
+  - name: roxctl-config
+    secret:
+      secretName: rox-api-token
```

Comment:
How it was loaded before?

Reply:
Before, the tasks created an environment variable in the container running the roxctl command, populated from the secret content.
In order to be able to provide the RHACS short-lived token obtained from the service account token to the task, a workspace is used, which can be backed either by a persistent volume, or by a secret.